apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-detail-page.component.html

```diff
@@ -9,6 +9,10 @@ <h1 class="page-title">
     <span class="text-muted">
       &nbsp;#{{vo.id}}
     </span>
+    <br>
+    <span class="vo-text">
+      {{vo.shortName}}
+    </span>
```

Comment:
tbh this looks weird. Try something like: (please check slack, couldnt it copy here)